sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/DependencyManager.groovy

```diff
@@ -94,11 +102,19 @@ class DependencyManager {
 
     /**
      * Triggers the project loading initially to have data in the service
+     * This is to be called after the view was initialized
      */
-    private void populateProjectService() {
-        LoadProjectsOutput output = new LoadProjectsPresenter(projectResourceService)
+    public void populateProjectService() {
+        LoadProjectsOutput output = new LoadProjectsPresenter(projectResourceService, notificationService)
         LoadProjectsInput loadProjects = new LoadProjects(loadProjectsDataSource, output)
         loadProjects.loadProjects()
     }
 
+    /**
+     * Returns the global notification center
+     * @return a notification center that handles app notifications
+     */
+    NotificationCenter getNotificationCenter() {
```

Comment:
```suggestion
    NotificationHandler getNotificationHandler() {
```

since it handles notification this might be a better name.

Reply:
you are right :+1: